### PR TITLE
Sticky Fields fix (Issue #29)

### PR DESCRIPTION
**Pull request by vb-cs**
This is a fix to issue #29. Selectively incorporates changes to participate.js made by @dckc that contribute to fixing the issue.

### CODE REVIEW

**Comment by dckc** [comment]
Made by me where? Remind me, please?

**Comment by vb-cs** [comment]
@dckc https://github.com/rchain-community/liquid-democracy/commit/3f979c2624fdc52f33ec042e4aeaa58a07467a7e

**Comment by dckc** [comment]
Ah. that was part of #11 ; I went ahead and merged that after merging #30 .

File: src/participate.js [hunk 1]
```diff
@@ -309,47 +309,47 @@ function actionControl(state, { html, getEthProvider }) {
...
             <input
               name=${name}
               value=${value}
               onchange=${(/** @type {Event} */ event) => {
-                state.fields = {
-                  ...state.fields,
-                  ...{ [name]: ckControl(event.target).value },
-                };
-              }}
+            const current = { [name]: ckControl(event.target).value };
```
Comment (Bill-Keuntje) [inline src/participate.js:342]: @dckc Is this the original fix to https://github.com/rchain-community/rgov/pull/190#issuecomment-868497204

Comment (dckc) [reply src/participate.js:342]: yes... well, it's hard to say whether the root cause of the symptoms reported in 190 are the same, but the symptoms are the same.

**Pull request closed without merging**